c++/src/kj/compat/http-test.c++

```diff
@@ -5166,16 +5186,16 @@ KJ_TEST("HttpClient WebSocket: client can have a custom WebSocket error handler"
     0xF0, 0x02, 'y', 'o'  // all RSV bits set, plus FIN
   };
   const byte closeFrame[] = {
-    0x88, 0xa8, 0xC, 0x22, 0x38, 0x4e, 0x3, 0xea, // FIN, opcode=Close, code=1009
-    'R', 'e', 'c', 'e', 'i', 'v', 'e', 'd', ' ',
-    'f', 'r', 'a', 'm', 'e', ' ',
-    'h', 'a', 'd', ' ',
-    'R', 'S', 'V', ' ',
-    'b', 'i', 't', 's', ' ',
-    '2', ' ',
-    'o', 'r', ' ',
-    '3', ' ',
-    's', 'e', 't',
+    0x88, 0xa8, 12, 34, 56, 78, 0x3^12, 0xea^34, // FIN, opcode=Close, code=1009
```

Comment:
Seems this test didn't realize that `0xC, 0x22, 0x38, 0x4e` was actually the fake mask value of `12, 34, 56, 78`. If we noticed we'd probably also have noticed that the payload wasn't masked...